src/Behat/Behat/Hook/Call/RuntimeScenarioHook.php

```diff
@@ -76,10 +76,6 @@ protected function isMatchTagFilter(FeatureNode $feature, ScenarioInterface $sce
     {
         $filter = new TagFilter($filterString);
 
-        if ($filter->isFeatureMatch($feature)) {
```

Comment:
It's a worry that this is just deletable 😅 

Makes sense if it's subsequently checked on the next line